base/server/etc/default.cfg

```diff
@@ -589,3 +615,51 @@ pki_import_shared_secret=False
 pki_share_db=True
 pki_share_dbuser_dn=uid=pkidbuser,ou=people,%(pki_ds_base_dn)s
 pki_source_phone_home_xml=/usr/share/pki/%(pki_subsystem_type)s/conf/phoneHome.xml
+
+pki_security_domain_setup=True
+pki_registry_enable=True
+
+[ACME]
+pki_ds_setup=False
+pki_security_domain_setup=False
+pki_registry_enable=False
+
+# Database params:
```

Comment:
Why these parameters are not  included with default empty value?

All parameters start with `pki_` while these starts with `acme_`. Actually I think start the subsystem variable with `<subsystem>_` and leave `pki_` to the global section could be better but I like consistency more.